.env.example

```
@@ -0,0 +1,31 @@
+PROJECT_NAME="Kaapi-Guardrails"
+STACK_NAME=Kaapi-Guardrails
+
+# API Base URL for cron scripts (defaults to http://localhost:8000 if not set)
+API_BASE_URL=http://localhost:8000
+
+# Postgres
+POSTGRES_SERVER=localhost
+POSTGRES_PORT=5432
+POSTGRES_DB=kaapi-guardrails
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_PORT key

(UnorderedKey)